geometry/robust_orientation.cpp

```diff
@@ -31,13 +31,40 @@ namespace m2 { namespace robust
     ASSERT_EQUAL ( res, true, () );
     UNUSED_VALUE(res);
 
-    double a[] = { p1.x, p1.y };
-    double b[] = { p2.x, p2.y };
+    // Change points order here, because predicates.c is written in the geodesic
```

Comment:
predicates.c confuses me. Does it mean a field c in a predicates struct, or there should be a space after the dot? What predicates this comment talks about?